packages/theming/apple-theme/src/createMacOSAliasTokens.ts

```diff
@@ -1,11 +1,10 @@
 import { getAliasTokens } from '@fluentui-react-native/theme-tokens';
 import { AliasColorTokens, AppearanceOptions } from '@fluentui-react-native/theme-types';
-import { memoize } from '@fluentui-react-native/memo-cache';
 import { mapPipelineToTheme } from '@fluentui-react-native/theming-utils';
 
 function createMacOSAliasTokensWorker(mode: AppearanceOptions): AliasColorTokens {
   const aliasTokens = getAliasTokens(mode);
   return mapPipelineToTheme(aliasTokens);
 }
 
-export const createMacOSAliasTokens = memoize(createMacOSAliasTokensWorker);
+export const createMacOSAliasTokens = createMacOSAliasTokensWorker;
```

Comment:
From my understanding of `memoize()`, it caches and returns the cached set of tokens if no changes are reflected. Since `Appearance` doesn't detect HC on macOS, the same set of tokens are returned when HC setting changes. So by removing memoize, we call getAliasTokens() on every theme change to handle this edge case.  In other words, without it getAliasTokens won't fire when HC changes

Reply:
Would it be possible to add `isHighContrast` or the like as an argument to `createMacOSAliasTokensWorker` so that it's properly reflected in the cache? I worry about the performance cost of recreating all the tokens / re-rendering everything by removing the `memoize()`.

I also wonder if for macOS, we can include checking `isHighContrast` into `AppearanceOptions`, similar to what's being done in [getCurrentAppearance.windows.ts](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/theming-utils/src/getCurrentAppearance.windows.ts). It looks like there's effort on that side to inlude HC as part of the react-native-windows `Appearance` API (https://github.com/microsoft/react-native-windows/issues/3701)

Reply:
Thanks for the suggestions, I added `memoize()` back in commit 510a5e7ca6193385b05f99f0e6e65a3503dc08a6.

Adding high contrast options to `AppearanceOptions` for macOS may be a bit tricky since it has light and dark HC modes... not sure if we want to incorporate platform specific stuff to cross-plat objects.